src/confcom/azext_confcom/README.md

```diff
@@ -0,0 +1,484 @@
+
+Example 4: The following command takes the input of an ARM template to create a human-readable CCE policy in clear text and print to console: <br />
+    
+    az confcom acipolicygen -a ".\arm-template.json" --outraw --print-policy
+
+Use `--outraw` argument to output policy in clear text compact REGO format.
+
+Example 5: Input an ARM template to create a human-readable CCE policy in pretty print REGO format and save the result to a file named ".\output-file.rego": <br />
+
+    az confcom acipolicygen -a ".\arm-template" --outraw-pretty-print --save-to-file ".\output-file.rego"
+
+Example 6: Validate the policy present in the ARM template under "ccepolicy" and the containers within the ARM template are compatible. If they are incompatible, a list of reasons is given and the exit status code will be 2: <br />
+
+    az confcom acipolicygen -a ".\arm-template.json" --diff
+
+Example 7: Decode the existing CCE policy in ARM template and print to console in clear text. 
+
+    az confcom acipolicygen -a ".\arm-template.json" --print-existing-policy --outraw
+
+Example 8: Generate a CCE policy using `--disable-stdio` argument. 
+`--disable-stdio` argument disables container standard I/O access by setting `allow_stdio_access` to false. 
+
+    az confcom acipolicygen -a ".\arm-template.json" --disable-stdio
+
+Example 9: Inject a CCE policy into ARM template. 
+This command adds the `--debug-mode` argument to enable executing /bin/sh and /bin/bash in the container group: <br />
+
+    az confcom acipolicygen -a .\sample-arm-input.json --debug-mode
+
+In the above example, The `--debug-mode` modifies the following to allow users to shell into the container via portal or the command line:
+
+1. Adds the following to container rule so that users can access bash process. 
+
+```
+   "exec_processes": [
+      {
+        "command": [
+          "/bin/sh"
+        ],
+        "signals": []
+      },
+      {
+        "command": [
+          "/bin/bash"
+        ],
+        "signals": []
+      }
+    ]
+```
+2. Changes the values of these three rules to true on the policy. 
+This is also for the purpose of allowing users to access logging, container properties and dump stack,  all of which are part of loggings as well. 
+See [A Sample Policy that Uses Framework](#a-sample-policy-that-uses-framework) for details for the following rules: 
+
+    - allow_properties_access
+    - allow_dump_stacks
+    - allow_runtime_logging
+    
+    
+Example 10: The confidential computing extension CLI is designed in such a way that generating policy does not necessarily have to depend on network calls as long as users have the layers of the images they want to generate policies for saved in a tar file locally. See the following example: <br />
+
+    docker save ImageTag -o file.tar
+
+Disconnect from network and delete the local image. 
+Use the following command to generate CCE policy for the image. 
```

Comment:
I would say "delete the local image from the docker daemon." since the tar file is technically also the "local image"